### PR TITLE
Add OpenTelemetry webpage to examples

### PR DESCRIPTION
**Pull request by pellared**
I think it is worth to add OpenTelemetry webpage as an example.
OpenTelemetry is the second most active CNCF project behind Kubernetes.

CC @chalin @svrnm @cartermp @austinlparker

### CODE REVIEW

**Review by chalin — APPROVED** [review]
❤️ 

I thought it was there, though maybe I'm recalling OTel appearing in a list from a Docsy quarterly report.

I'll let @LisaFC or @emckean merge this if they agree with the update :).

**Comment by LisaFC** [comment]
No, it's not there already, this is great!

**Pull request merged**
